### PR TITLE
Magento v2

### PR DESCRIPTION
**Pull request by deanfeb**
fix autoload dan readme

### CODE REVIEW

File: magentov2/README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
-# indodana paylater payment method module for magento >= 2.3.x 
+# indodana paylater payment method module for magento 2.4.0 >= 2.3.x
```
Comment (stvyang) [inline magentov2/README.md:1]: Is `2.4.0` here intended?

File: magentov2/README.md [hunk 2]
```diff
@@ -11,6 +19,27 @@
...
+    3. Copy plugin/app/ to <magento root>/app/
+    4. Set folder Acces Permission 
+        cd <magento_root>
+        find var generated vendor pub/static pub/media app/etc -type f -exec chmod u+w {} +
+        find var generated vendor pub/static pub/media app/etc -type d -exec chmod u+w {} +
+        chmod u+x bin/magento
+    5. Run <magento_root>$sudo bin/magento setup:upgrade
+    6. Run <magento_root>$sudo bin/magento module:enable Indodana_PayLater
+    7. Run <magento_root>$sudo bin/magento setup:upgrade
+    8. Jika mode developer skip bagian ini ( coba skip dulu )
```
Comment (stvyang) [inline magentov2/README.md:35]: Please use English

File: magentov2/plugin/app/code/Indodana/PayLater/Controller/Index/cancel.php [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <?php
 
 namespace Indodana\PayLater\Controller\Index;
+require_once( dirname(dirname(dirname(__FILE__))) . '/vendor/autoload.php');
```
Comment (stvyang) [inline magentov2/plugin/app/code/Indodana/PayLater/Controller/Index/cancel.php:4]: Add newline above and below line 4

**Review by stvyang — CHANGES_REQUESTED** [review]

File: magentov2/README.md (hunk 2 above)
Comment (deanfeb) [reply magentov2/README.md]: done

File: magentov2/plugin/app/code/Indodana/PayLater/Controller/Index/cancel.php (hunk 3 above)
Comment (deanfeb) [reply magentov2/plugin/app/code/Indodana/PayLater/Controller/Index/cancel.php:5]: done

File: magentov2/README.md (hunk 1 above)
Comment (deanfeb) [reply magentov2/README.md:1]: you can use version 2.3.x

Comment (deanfeb) [reply magentov2/README.md]: already changes to # indodana paylater payment method module for magento 2.4.0 or magento 2.3.x

**Pull request merged**